ecosystem.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const pkg = require('./package.json');
+
+const cwd = process.cwd();
```

Comment:
What's this file's purpose?

Reply:
it's a PM2 configuration stub

Reply:
A makes sense thank you, I should add that to https://github.com/NewCircuit/bot-template